automation/taskcluster/lib/gradle.py

```diff
@@ -29,7 +29,7 @@ def get_geckoview_versions():
     output = _run_gradle_process('printGeckoviewVersions')
 
     versions = {}
-    for version_type in ('beta',):
+    for version_type in ('release',):
```

Comment:
We could remove this loop until we have multiple version types :)

Reply:
Good call, done!